include/openenclave/internal/calls.h

```diff
@@ -460,9 +460,15 @@ oe_result_t oe_ecall(
  * not whether it was successful. The ECALL implementation must define its own
  * error reporting scheme based on its parameters.
  *
+ * The argument description parameters (*_size and *_is_pointer) are only used
```

Comment:
remove "only used for OP-TEE" since there may be other TEEs in the future that need them too and it's easy to miss this comment when adding another TEE.

Reply:
Perhaps this should read "GlobalPlatform-compliant TEEs". That's really what we are adding support for in the SDK; the fact that the only GP-compliant TEEs work on ARM is orthogonal.

Reply:
Rephrased.